5_recognition.ipynb

```diff
@@ -238,25 +425,50 @@
     "HSVは、色相(Hue)、彩度(Saturation)、明度(Value)によって色を表現します。今回は色相情報のみを利用してレゴブロックの水色を抽出します。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "The color of the Lego block is \"light blue\", so let's use the color to detect the Lego block.\n",
```

Comment:
I have a question about "the Lego block".
Is it just one Lego block or is it the case that there is more than one Lego block?

Reply:
Good point.

We have multiple Lego blocks with each different color.
One of the block is "light blue".

Reply:
Ah, I see.
So in this case we are referring to a specific one from the set.
So, "the Lego block" works:

    "The color of the Lego block is \"light blue\", so let's use the color to detect the Lego block.\n",